lib/modules/manager/poetry/update-locked.ts

```diff
@@ -0,0 +1,23 @@
+import { logger } from '../../../logger';
+import type { UpdateLockedConfig, UpdateLockedResult } from '../types';
+import { extractLockFileEntries } from './locked-version';
+
+export function updateLockedDependency(
+  config: UpdateLockedConfig
+): UpdateLockedResult {
+  const { depName, currentVersion, newVersion, lockFile, lockFileContent } =
+    config;
+  logger.debug(
+    `poetry.updateLockedDependency: ${depName}@${currentVersion} -> ${newVersion} [${lockFile}]`
+  );
+  try {
+    const locked = extractLockFileEntries(lockFileContent || '');
+    if (depName && locked[depName] === newVersion) {
+      return { status: 'already-updated' };
+    }
+    return { status: 'unsupported' };
+  } catch (err) /* istanbul ignore next */ {
```

Comment:
I think we don't need the try catch. I don't see a place where code throws. Lockfile parse errors are already catched.